cmd/kpod/README.md

```diff
@@ -0,0 +1,16 @@
+# kpod - Simple debugging tool for pods and images
+kpod is a simple client only tool to help with debugging issues when daemons such as CRI runtime and the kubelet are not responding or
+failing. A shared API layer could be created to share code between the daemon and kpod. kpod does not require any daemon running. kpod
+utilizes the same underlying components that ocid uses viz., containers/image, container/storage, oci-runtime-tool/generate, runc or
```

Comment:
I have never seen it before either,   But Wikipedia says it makes sense.

https://en.wikipedia.org/wiki/Viz.